README.md

```diff
@@ -1,111 +1,50 @@
+          , module Colog.Core.IO
+          , module Colog.Core.Severity
+          ) where
 
-* [Intro: Using `LogAction`](https://github.com/kowainik/co-log/blob/main/co-log/tutorials/1-intro/Intro.md)
-* [Using custom monad that stores `LogAction` inside its environment](https://github.com/kowainik/co-log/blob/main/co-log/tutorials/2-custom/Custom.md)
-
-`co-log` also cares about concurrent logging. For this purpose we have the `concurrent-playground`
-executable where we experiment with different multithreading scenarios to test the library's behavior.
-You can find it here:
-
-* [tutorials/Concurrent.hs](co-log/tutorials/Concurrent.hs)
-
-## Benchmarks
-
-`co-log` is compared with basic functions like `putStrLn`. Since IO overhead is
-big enough, every benchmark dumps 10K messages to output. If a benchmark's name
-doesn't contain `Message` then this benchmark simply dumps the string `"message"`
-to output, otherwise it works with the `Message` data type from the `co-log`
-library.
-
-To run benchmarks, use the following command:
-
-```
-cabal v2-run co-log-bench
-```
-
-| Benchmarks                                              | Time for 10K messages |
-| :------------------------------------------------------ | :-------------------- |
-| `Prelude.putStrLn`                                      | `  5.117ms`           |
-| `Text.putStrLn`                                         | `  9.220ms`           |
-| `ByteString.putStrLn`                                   | `  2.971ms`           |
-| `mempty`                                                | `  1.181ms`           |
-| `logStringStdout`                                       | `  5.107ms`           |
-| `logPrint`                                              | `  5.248ms`           |
-| `logTextStdout`                                         | `  5.351ms`           |
-| `logByteStringStdout`                                   | `  2.933ms`           |
-| `logByteStringStderr`                                   | ` 17.482ms`           |
-| `ByteString > (stdout <> stderr)`                       | ` 17.715ms`           |
-| `Message > format > stdout`                             | `  9.188ms`           |
-| `Message > format > ByteString > stdout`                | `  3.524ms`           |
-| `Message{callstack} > format > stdout`                  | `  9.139ms`           |
-| `Message{callstack:5} > format > stdout`                | `  9.464ms`           |
-| `Message{callstack:50} > format > stdout`               | `  9.439ms`           |
-| `Message{Time,ThreadId} > format > stdout`              | ` 54.160ms`           |
-| `Message{Time,ThreadId} > format > ByteString > stdout` | ` 54.137ms`           |
-
-
-[hk-img]: https://img.shields.io/hackage/v/co-log.svg?logo=haskell
-[hk-img-ps]: https://img.shields.io/hackage/v/co-log-polysemy.svg?logo=haskell
+    import Colog.Core.Action
+    import Colog.Core.Class
+    import Colog.Core.IO
+    import Colog.Core.Severity 
+    ```
```

Comment:
I would write the following simple example:

```haskell
module Main (main) where

import Prelude hiding (log)

import Colog.Core (LogAction, logStringStdout, (<&))


app :: LogAction IO String -> IO ()
app log = do
    log <& "Starting app..."
    log <& "Finishing app..."

main :: IO ()
main = app logStringStdout
```

I haven't checked if it compiles, so, please, could you check if everything is good and fix any errors if there's anything?

Reply:
I tried compiling and got an error. This error might be due to my mistake.

[1 of 1] Compiling Main             ( Main.hs, Main.o )

Main.hs:5:1: error:
    Could not find module ‘Colog.Core’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
  |
5 | import Colog.Core (LogAction, logStringStdout, (<&))
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reply:
@udaycruise2903 How exactly do you compile the example?

Reply:
i compiled this example in my local system with command - GHC Main

Reply:
Ah, yes, this won't work because `ghc` doesn't know about the `co-log-core` package that's why it can't find the module.

One way to check the example could be a start a repl with `cabal`, bringing `co-log-core` in scope and trying to load the module:

```
$ cabal repl -b co-log-core
ghci> :l Main.hs
```

Reply:
five days ago,  i had tried testing example with cabal, i dont remember results. 
i tried again and the current error is -
```
[neo@nebula co-log-core]$ cabal repl -b co-log-core
Resolving dependencies...
Error:
    Dependency cycle between the following components: library
    In the inplace package 'co-log-core-0.3.0.0'
```

Reply:
Haskell tooling gives us hard times... Probably this happens because you're running this command from the `co-log-core` repository and cabal conflicts local `co-log-core` with the one from Hackage. I think you can either try running this command outside of the package directory or just with `cabal repl` to bring the local package into the scope.

Reply:
Co-log-core and the example you have mentioned works fine🙂. Thanks for guiding me through this issue🙏.
```
[neo@nebula ~]$ cabal repl -b co-log-core
Resolving dependencies...
Build profile: -w ghc-8.10.7 -O1
In order, the following will be built (use -v for more details):
 - co-log-core-0.3.0.0 (lib) (requires download & build)
 - fake-package-0 (lib) (first run)
Downloading  co-log-core-0.3.0.0
Downloaded   co-log-core-0.3.0.0
Starting     co-log-core-0.3.0.0 (lib)
Building     co-log-core-0.3.0.0 (lib)
Installing   co-log-core-0.3.0.0 (lib)
Completed    co-log-core-0.3.0.0 (lib)
Configuring library for fake-package-0..
Preprocessing library for fake-package-0..
Warning: No exposed modules
GHCi, version 8.10.7: https://www.haskell.org/ghc/  :? for help
Loaded GHCi configuration from /tmp/cabal-repl.-1843/setcwd.ghci
Prelude> :l ~/Documents/github/co-log-core/example/Main.hs
[1 of 1] Compiling Main             ( /home/neo/Documents/github/co-log-core/example/Main.hs, interpreted )
Ok, one module loaded.
*Main> 
```